### DISCUSSION TITLE
May I ask why in versions 2.15 and 2.13, the `client-go` is at version `0.29.4`, but here 2.14 it has changed to version `1.5.2`?

### DISCUSSION THREAD

**Discussion by y805939188** [kedacore/keda#6025] [Q&A / Need Help] [answered]
Hi. May I ask why in versions 2.15 and 2.13, the client-go is at version 0.29.4, but here 2.14 it has changed to version 1.5.2?
I upgraded my KEDA version in cluster, but when I upgraded the version in my code, I found that `client-go` was not compatible.

In release/2.13:
<img width="624" alt="image" src="https://github.com/user-attachments/assets/cbef67e5-c118-4ae0-b181-2340613ddbcc">
In release/2.14:
<img width="577" alt="image" src="https://github.com/user-attachments/assets/042c5ee3-20c3-4fa5-b394-e1ace7e4db09">
In release/2.15:
<img width="582" alt="image" src="https://github.com/user-attachments/assets/ceda4413-0506-47bc-85bf-115b3e37ba48">

**Comment by JorTurFer** [comment, marked as answer]
Probably we bumped it by mistake, but it doesn't change anything else from the go mod, as there is a replacement a few lines below, pinning the version v0.28.9 -> 
![image](https://github.com/user-attachments/assets/e201d1a0-cd04-48e0-9b77-bff072ae5b41)

Basically, setting there 0.29. 1.5.2 or 26.234.63 doesn't make the difference if the `replace` hasn't changed

**Reply by y805939188** [reply]
Thank you answer.
😂 😂 😂 Actually, the reason I encountered this issue is that I am using KEDA 2.14 in a cluster, and I want to integrate KEDA into my code. So, I installed github.com/kedacore/keda/v2@2.14, but it keeps updating my client-go version, which causes some dependencies to be incompatible. Nevertheless, I switched to install v2.15 in my code, and it seems that everything is fine now.
Thanks again~
